src/main/java/com/gtnewhorizons/angelica/AngelicaMod.java

```diff
@@ -15,7 +15,7 @@
         name = "Angelica",
         version = Tags.VERSION,
         dependencies = " before:lwjgl3ify@[1.5.3,);" + " after:hodgepodge@[2.4.4,);" + " after:CodeChickenCore@[1.2.0,);"
-                     + " after:archaicfix@[0.7.0,);",
+                     + " after:archaicfix@[0.7.0,);" + " after:gtnhlib@[0.2.1,);",
```

Comment:
this should be a `required-after`; the other mods are optional and if loaded needed to be that version; angelica now won't function without NH lib

Reply:
```suggestion
                     + " after:archaicfix@[0.7.0,);" + " required-after:gtnhlib@[0.2.1,);",
```

Reply:
Yeah makes sense. I'm not familiar with the dependency string format